modules/modal/css/README.md

```diff
@@ -0,0 +1,89 @@
+# Canvas Kit Modal
+
+A Modal component that allows you to render a Popup with a translucent overlay.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-css
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-css-modal
+```
+
+Add your `node_modules` directory to your SASS `includePaths`. You will then be able to import
+`index.scss`.
+
+```scss
+@import '~@workday/canvas-kit-css-modal/index.scss';
+```
+
+## Usage
+
+Use `.wdc-modal-bg` to create the background overlay and `.wdc-modal` to create the modal itself.
+The title and body content can be styled using `.wdc-modal-title` and `.wdc-modal-body`,
+respectively.
+
+```html
+<div class="wdc-modal-bg">
+  <div class="wdc-modal">
+    <div class="wdc-modal-title">Modal Title</div>
+    <div class="wdc-modal-body">Modal content</div>
+  </div>
+</div>
+```
+
+### With Close Button
+
+```jsx
+import {xIcon} from '@workday/canvas-system-icons-web';
+import {SystemIcon} from '@workday/canvas-kit-react';
+
+<div className="wdc-modal-bg">
+  <div className="wdc-modal">
+    <div className="wdc-modal-close">
+      <button onClick={this.onCloseClick} className="wdc-btn-icon-plain">
+        <SystemIcon icon={xIcon} />
```

Comment:
Can we update this to use `<i className="wdc-icon" data-icon="xIcon" data-category="system" />` and update the example to be plain html?

Reply:
Sure, I'll update the example in the `README` -- should I need to update the code in `stories.tsx` as well?

Reply:
I'm less concerned about the stories because they're already in jsx, but we might as well so they match

Reply:
Sounds good, the buttons will revert to their non-beta styles then since I don't think we have the beta buttons in CSS yet...?

Reply:
Never mind, I realize you were probably just referring to updating the story code for the `xIcon` to be html and not the entire story itself. I left the beta buttons in.